version.props

```diff
@@ -1,7 +1,7 @@
 <Project>
   <PropertyGroup>
-    <VersionPrefix>2.2.0</VersionPrefix>
-    <VersionSuffix>preview1</VersionSuffix>
+    <VersionPrefix>3.0.0</VersionPrefix>
+    <VersionSuffix>alpha1</VersionSuffix>
```

Comment:
Any reason we picked alpha1 instead of preview1? I was under the impression we didn't do "alpha" releases.

Reply:
yes. Intentionally chose alpha1 for now because when we move official builds to VSTS we'll likely be changing build number schema. I wanted to make sure we don't start producing builds now which are "newer" (nuget version) than what we will produce in a few weeks.